identity/src/main/java/com/stripe/android/identity/ui/DebugScreen.kt

```diff
@@ -179,7 +289,39 @@ private fun PreviewUserExperienceSection(
     }
 }
 
-internal const val TEST_TAG_COMPLETE_BUTTON = "Completed"
+@Composable
+private fun CompleteOptionRow(
+    content: String,
+    selected: Boolean,
+    enabled: Boolean,
+    testTag: String,
+    onClick: () -> Unit
+) {
+    Row(modifier = Modifier.fillMaxWidth(), verticalAlignment = Alignment.CenterVertically) {
+        RadioButton(
+            selected = selected,
+            enabled = enabled,
+            modifier = Modifier.testTag(testTag),
+            onClick = onClick
+        )
+        StyledClickableText(
+            text = AnnotatedString(content),
+            modifier = Modifier.padding(start = 8.dp),
+            enabled = enabled,
+            onClick = { onClick() }
+        )
```

Comment:
should we instead make the entire row clickable? And use a regular text here (so that we don't have to maintain a `StyledClickableText`)

Reply:
Good idea, just removed!